reactor-core/src/main/java/reactor/core/publisher/ContextPropagation.java

```diff
@@ -49,7 +49,8 @@ final class ContextPropagation {
 
 	static final Logger LOGGER;
 
-	static final boolean isContextPropagationAvailable;
+	// Note: If a reflection is used for this field, then the name of the field should end with 'Available'
```

Comment:
```suggestion
	// Note: If reflection is used for this field, then the name of the field should end with 'Available'
```

Reply:
It would be also great to have an answer to "why" - is this processing based on variable name suffix documented anywhere?

Reply:
No this is not documented and the functionality is located in Spring Framework.
The functionality in Spring Framework, related to Reactor projects, is a short term solution. Ultimately the long term solution should be in each Reactor project that needs such.

Reply:
Thanks for the clarification and adding the comment!